tests/phpstan/bootstrap.php

```diff
@@ -2,12 +2,21 @@
 /*
  * Global constants used in the analysed code need to be defined in bootstrap files.
  */
-
-define( 'WEBSTORIES_VERSION', '0.0.0' );
-define( 'WEBSTORIES_DB_VERSION', '0.0.0' );
-define( 'WEBSTORIES_AMP_VERSION', '0.0.0' );
-define( 'WEBSTORIES_PLUGIN_FILE', dirname( __DIR__, 2 ) . '/web-stories.php' ); // phpcs:ignore PHPCompatibility.FunctionUse.NewFunctionParameters.dirname_levelsFound
-define( 'WEBSTORIES_PLUGIN_DIR_PATH', dirname( WEBSTORIES_PLUGIN_FILE ) );
+if ( ! defined( 'WEBSTORIES_VERSION' ) ) {
+	define( 'WEBSTORIES_VERSION', '0.0.0' );
+}
+if ( ! defined( 'WEBSTORIES_DB_VERSION' ) ) {
+	define( 'WEBSTORIES_DB_VERSION', '0.0.0' );
+}
+if ( ! defined( 'WEBSTORIES_AMP_VERSION' ) ) {
+	define( 'WEBSTORIES_AMP_VERSION', '0.0.0' );
+}
+if ( ! defined( 'WEBSTORIES_PLUGIN_FILE' ) ) {
+	define( 'WEBSTORIES_PLUGIN_FILE', dirname( __DIR__, 2 ) . '/web-stories.php' ); // phpcs:ignore PHPCompatibility.FunctionUse.NewFunctionParameters.dirname_levelsFound
+}
+if ( ! defined( 'WEBSTORIES_PLUGIN_DIR_PATH' ) ) {
+	define( 'WEBSTORIES_PLUGIN_DIR_PATH', dirname( WEBSTORIES_PLUGIN_FILE ) );
+}
```

Comment:
If you remove `tests/phpunit/unit/bootstrap.php` from the config file then we can undo all of these changes here